content/testnet.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Setup Rubix Node and Joining the Testnet
+aliases:
+  [
+    "/setup",
+    "/rbt",
+    "/dist",
+    "/start",
+  ]
+geekdocBreadcrumb: false
+weight: 1
+---
+
+Faucet tokens are premined tokens available for users to test and interact with the Rubix testnet.
+
+When a new node joins the testnet, it will have 0 tokens to start with. To help users get started, the Rubix testnet provides a faucet service where users can request premined test tokens. These tokens are not earned through validation but are instead distributed for testing and development purposes.
+
+To request test tokens, users must first create a node in the Rubix testnet and generate a Decentralized Identifier (DID). The DID is then submitted to the faucet, which allocates a fixed number of test tokens to the requesting node.
```

Comment:
Faucet don't "allocate" , faucet transfer requested amount of tokens to the users DID.  The faucet have kept a limit on amount of token per node inorder to avoid spam requests.

Reply:
Okay, will update the words.